tests/epics/pmac/test_pmac.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from scanspec.specs import Line, fly
+
+from ophyd_async.core import DeviceCollector, set_mock_value
+from ophyd_async.epics.pmac import PmacCSMotor, PmacTrajectory
+
+
+@pytest.fixture
+async def sim_x_motor():
+    async with DeviceCollector(mock=True):
+        sim_motor = PmacCSMotor(
+            "BLxxI-MO-STAGE-01:X", "BRICK1.CS3", "z", name="sim_x_motor"
+        )
+
+    set_mock_value(sim_motor.motor_egu, "mm")
+    set_mock_value(sim_motor.precision, 3)
+    set_mock_value(sim_motor.acceleration_time, 0.5)
+    set_mock_value(sim_motor.max_velocity, 5)
+    set_mock_value(sim_motor.velocity, 0.5)
+
+    yield sim_motor
+
+
+async def test_sim_pmac_simple_trajectory(sim_x_motor) -> None:
+    # Test the generated Trajectory profile from a scanspec
+    async with DeviceCollector(mock=True):
+        prefix = "BLxxI-MO-STEP-01"
+        motors = [sim_x_motor]
+        traj = PmacTrajectory(prefix, "BRICK1.CS3", motors, name="sim_pmac")
```

Comment:
This is the only line we need inside the DeviceCollector, the rest can be dedented